cockroachcloud/export-logs.md

```diff
@@ -129,21 +129,79 @@ Perform the following steps to enable log export from your {{ site.data.products
+        {% include_cached copy-clipboard.html %}
+        ~~~shell
+        curl --request POST \
+          --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/logexport \
+          --header "Authorization: Bearer {secret_key}" \
+          --data '{"type": "AWS_CLOUDWATCH", "log_name": "{log_group_name}", "auth_principal": "{role_arn}"}'
+        ~~~
+
+        Where:
+        - `{cluster_id}` is your {{ site.data.products.dedicated }} cluster ID as determined in step 3.
+        - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for instructions on generating this key.
+        - `{log_group_name}` is the target AWS CloudWatch log group you created in step 1.
+        - `{role_arn}` is the ARN for the `CockroachCloudLogExportRole` role you copied in step 8.
+
+    1. To enable log export for your {{ site.data.products.dedicated }} cluster with custom logging configuration:
+
+        1. Consult the log export entry on the [Cockroach Cloud API Reference](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters/-cluster_id-/logexport) and select the **Schema** tab to view the supported log configuration options, and determine the customized logging configuration you would like to use.
+
+            For example, consider the following configuration:
+
+            {% include_cached copy-clipboard.html %}
+            ~~~json
+            {
+             type: AWS_CLOUDWATCH,
+             log_name: "default",
+             auth_principal: "{role_arn}",
+             redact: true,
+             region: "",
+             groups: [
+                     {
+                         log_name: "sql",
+                         channels: ["SQL_SCHEMA", "SQL_EXEC"],
+                         redact: false,
+                     },
+                     {
+                         log_name: "devops",
+                         channels: ["OPS", "HEALTH", "STORAGE"]
+                         min_level: "WARNING"
+                     },
+             ]
+            }
+            ~~~
+
+            This configuration:
+            - Enables [redaction](/docs/{{site.versions["stable"]}}/configure-logs.html#redact-logs) globally for all log entries emitted to AWS CloudWatch.
+            - Sends log entries in the `SQL_SCHEMA` and `SQL_EXEC` [logging channels](/docs/{{site.versions["stable"]}}/logging-overview.html#logging-channels) to a AWS CloudWatch log group named `sql`, and overrides (disables) the global redaction configuration for just these two log channels only.
+            - Sends log entries in the `OPS`, `HEALTH`, and `STORAGE` [logging channels](/docs/{{site.versions["stable"]}}/logging-overview.html#logging-channels) to an AWS CloudWatch log group named `devops`, but only for those entries that are of log [severity level](/docs/{{site.versions["stable"]}}/logging.html#logging-levels-severities) `WARNING` or higher.
```

Comment:
didn't see it anywhere but we should mention that the rest of the channels get sent to the default sink identified at the top level of the JSON